crates/nu-protocol/src/errors/shell_error.rs

```diff
@@ -1330,14 +1330,15 @@ This is an internal Nushell error, please file an issue https://github.com/nushe
         span: Span,
     },
 
-    #[error("Deprecated: {old_command}")]
-    #[diagnostic(help("for more info see {url}"))]
```

Comment:
Did we ever satisfy that with useful URLs :P ? 
Slightly worried with those half sentence format strings, when folks may treat that as free text and never check the format string but deprecation warnings shouldn't live too long.